rs/ethereum/cketh/test_utils/src/lib.rs

```diff
@@ -594,9 +594,9 @@ impl CkEthSetup {
 
     pub fn max_logs_block_range(&self) -> u64 {
         if self.evm_rpc_id.is_none() {
-            799
+            798
         } else {
-            500
+            499
```

Comment:
Note that none of the integration tests were changed, ensuring that we have the same behaviour regarding log scraping. The reason for this change is due to the unclarity of RPC providers, e.g. Cloudflare states
> Limitations: Max block range of 800 blocks.

The question is then, is this inclusive or not? It turns out for Cloudflare it's not (the range should contain at most 799 blocks) but for Ankr it is, so the new implementation is a bit more conservative there.

Reply:
Good catch!